konfera/tests/test_models.py

```diff
@@ -133,6 +133,28 @@ def test_string_representation(self):
             '%s %s %s' % (dict(TITLE_CHOICES)[entry.title], entry.first_name, entry.last_name)
         )
 
+    def test_automatic_order_generator(self):
+        location = models.Location(title='test_title', street='test_street', city='test_city', postcode=000000,
+                                   state='test_state', capacity=20)
+        location.save()
+        event = models.Event(title='test_event', description='test', event_type='meetup', status='published',
+                             location=location, date_from=timezone.now(), date_to=timezone.now())
```

Comment:
you can store timezone.now() in variable and use it everywhere. Reason for this is that you pre-cache the value so you don't need to ask for it multiple times, and other thing is that the values would be different as milliseconds would change while this will be processed.